src/helm.documentProvider.ts

```diff
@@ -78,6 +79,16 @@ export class HelmInspectDocumentProvider implements vscode.TextDocumentContentPr
             } else if (uri.authority === helm.INSPECT_REPO_AUTHORITY) {
                 const id = uri.path.substring(1);
                 const version = uri.query;
+
+                if (!shell.isSafe(id)) {
+                    vscode.window.showWarningMessage(`Unexpected characters in chart name ${id}. Use Helm CLI to inspect this chart.`);
```

Comment:
Should this report what the offending characters were in the id?

Reply:
That sounds more effort than I'm interested in putting in for a "should never happen" scenario.  _grin_  I can certainly do this but it's a bit fiddly and verbose and I'm not sure it's worth it unless we actually hit this happening in the wild.

(FX: karma spinning up to hit it on day one after release.)